apps/web-mzima-client/src/app/shared/components/search-form/search-form.component.scss

```diff
@@ -687,3 +687,15 @@
     }
   }
 }
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap; /* Added for accessibility */
+  border: 0;
```

Comment:
Do you think this could be useful as a general style? If so, I suggest you move the class to styles.scss 👍

Reply:
That is outdated - following this discovery here : https://github.com/ushahidi/platform-client-mzima/pull/830#issuecomment-2008955786. Git pull the recent changes.